### ISSUE TITLE
Limit for 'multiple-ranged'

### ISSUE THREAD

**Issue by kudrin** [uvarov-frontend/vanilla-calendar-pro#303]
I would like to request the ability to set a minimum and maximum limit for date selections when using the multiple-ranged selection mode. This feature would be extremely useful in scenarios where a service or rental is available only for a specific duration. For example, an apartment that does not offer daily cleaning might only be rentable for a minimum of 3 days and a maximum of 7 days.

**Comment by ghiscoding** [comment]
have you tried [`settings.range.min`](https://vanilla-calendar.pro/docs/reference/additionally/settings#range-min)?

**Comment by uvarov-frontend** [comment]
Я так понял вам нужно установить минимальную и максимальную дату после того как пользователь кликнул на первую дату? Если да то это можно реализовать с помощью `actions`, например отследите первый клик с помощью `clickDay` и если в `selectedDate` есть первый день то вы полните свою логику по установке `settings.range.min` и `settings.range.max`, затем выполните `update`.

Посмотрите этот код и улучшите его для себя:
```
{
  settings: {
    selection: {
      day: "multiple-ranged",
    },
  },
  actions: {
    clickDay(event, self) {
      if (!self.selectedDates[0]) return;
      self.settings.range.min = self.selectedDates[0];
      // self.settings.range.max = ""; // your any date
      self.update();
    },
  },
}
```
Sandbox example: https://codesandbox.io/p/sandbox/priceless-dijkstra-np4zhj?file=%2Fcalendar.ts

**Comment by kudrin** [comment]
Good solution, but it is not quite user-friendly. Because after I have selected a date, I will not be able to select what is outside the range. I would prefer that the limits work with the on-hover action.

I have forked your project and [realize PR](https://github.com/kudrin/vanilla-calendar-pro/pull/1/files) that is how I see it. My frontend developer skills leave much to be desired, so I did not start doing this in the main branch.
I also attach screencast how it look like.

[Screencast from 2024-09-23 13-42-54.webm](https://github.com/user-attachments/assets/8a35478b-60b2-46b6-ab04-15915012a4fd)
